.github/workflows/release.yml

```diff
@@ -14,7 +15,7 @@ jobs:
       fail-fast: true
       matrix:
         node:
-          - 18
+          - 20
```

Comment:
Yeah I think this can pretty reasonably be any version we want for the release workflow